recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - if not exist %SCRIPTS% mkdir %SCRIPTS%
+    - xcopy /i /v /f %RECIPE_DIR%\\vs2008_patch\\setup_x64.bat %SCRIPTS%\\
+    - xcopy /i /v /f /s %RECIPE_DIR%\\vs2008_patch\\x64 %SCRIPTS%\\x64\\
+
+test:
+  commands:
+    - IF NOT EXIST %SCRIPTS%\\setup_x64.bat exit 1
+    - IF NOT EXIST %SCRIPTS%\\x64 exit 1
+
+about:
+  home: https://github.com/conda-forge/vs2008_express_x64_patch-feedstock
+  summary: Workaround to help CMake find the appropriate x64 compiler on VS 2008 Express with VS compiler for python on top.
+  license: BSD 3-Clause
+
+extra:
+  recipe-maintainers:
+    - msarahan
+    - patricksnape
```

Comment:
I suppose. 😢

To be honest though, I don't expect this to see lots of activity. It basically works. @patricksnape put a lot of working in getting this put together right.